SECURITY.md

```diff
@@ -32,4 +32,4 @@ We prefer all communications to be in English.
 
 Microsoft follows the principle of [Coordinated Vulnerability Disclosure](https://www.microsoft.com/en-us/msrc/cvd).
 
-<!-- END MICROSOFT SECURITY.MD BLOCK -->
\ No newline at end of file
+<!-- END MICROSOFT SECURITY.MD BLOCK -->
```

Comment:
Could you please revert this file change?

Reply:
I don't know why that is coming up as a change, there's no rich difference

Reply:
It could be a weird line ending change, but we should revert it before we can merge the PR! You could perhaps rebase and drop this file completely from the PR to fix this.

Reply:
Thanks!